modules/gcp_bigquery_table/outputs.tf

```diff
@@ -3,5 +3,5 @@ output "self_link" {
 }
 
 output "id" {
```

Comment:
this output is not explanatory, please improve the name. Also let's add a description to outputs please.

Reply:
Done